plugins/org.jkiss.dbeaver.ext.exasol/src/org/jkiss/dbeaver/ext/exasol/model/ExasolCurrentUserPrivileges.java

```diff
@@ -62,6 +62,10 @@ public Boolean getatLeastV5() {
     public Boolean getatLeastV6() {
         return majorVersion >= 6;
     }
+
+    public Boolean getatLeastV8() {
```

Comment:
```suggestion
    public Boolean getAtLeastV8() {
```

Reply:
Thanks @E1izabeth , should be fixed now.